application/src/Api/Adapter/MediaAdapter.php

```diff
@@ -200,4 +201,16 @@ public function preprocessBatchUpdate(array $data, Request $request)
 
         return $data;
     }
+
+    public function getFulltextText($resource)
+    {
+        $renderer = $this->getServiceLocator()
+            ->get('Omeka\Media\Renderer\Manager')
+            ->get($resource->getRenderer());
+        $mediaText = '';
+        if ($renderer instanceof FulltextSearchableInterface) {
+            $mediaText = $renderer->getFulltextText($this->getRepresentation($resource));
+        }
+        return sprintf('%s %s', parent::getFulltextText($resource), $mediaText);
```

Comment:
This is really minor but: this introduces an extra space at the end of the fulltext when the renderer isn't fulltext-searchable or has no text.